src/Client.js

```diff
@@ -610,6 +610,43 @@ class Client extends EventEmitter {
         }, chatId);
     }
 
+    /**
+     * Enables and returns the pin state of the Chat
+     * @returns {boolean}
+     */
+    async pinChat(chatId) {
+        return await this.pupPage.evaluate(async chatId => {
```

Comment:
```suggestion
        return this.pupPage.evaluate(async chatId => {
```

https://eslint.org/docs/rules/no-return-await I'm going to enable this rule soon